frontend/src/app/shared/models/user-solution/user-code.ts

```diff
@@ -0,0 +1,5 @@
+export interface UserCode {
```

Comment:
```suggestion
export interface IUserCode {
```

Reply:
This interface turned out to be redundant, so we removed it